pkg/cmd/server/api/validation/master.go

```diff
@@ -192,5 +192,12 @@ func ValidatePolicyConfig(config api.PolicyConfig) fielderrors.ValidationErrorLi
 func ValidateProjectRequestConfig(config api.ProjectRequestConfig) fielderrors.ValidationErrorList {
 	allErrs := fielderrors.ValidationErrorList{}
 
+	if len(config.ProjectRequestTemplate) > 0 {
```

Comment:
ProjectRequestTemplate is optional (https://github.com/openshift/origin/pull/2011/files#diff-381e09db7a55a712234f2c2413f7d987R92), but if you specify it, it must be well-formed.